### ISSUE TITLE
move Fortress to the userfrosting organization

### ISSUE THREAD

**Issue by alexweissman** [alexweissman/fortress#4]
Although Fortress can be used as a standalone library, it is designed very much with UserFrosting in mind.  I believe it should be moved under the umbrella of the [userfrosting](https://github.com/userfrosting) organization.

**Comment by frostbitten** [comment]
Yes. I was under the impression you would be maintaining this for the sake of multiple frameworks but since it is primarily for UF I have to agree. Also, moving to UF should increase the community involvement

**Comment by alexweissman** [comment]
Yup.  The only thing that will really be changing is the name - it will still be available on Packagist.
